wiki/Beatmapping/Health_Drain/en.md

```diff
@@ -0,0 +1,7 @@
+# Health drain
+
+Health drain refers to the natural drain of the health bar that occurs when playing a [beatmap](/wiki/Beatmaps) while disregarding the influence of [hit objects](/wiki/Hit_Objects). This drain is dependent on the HP difficulty setting, which is set by the mapper of a beatmap. This value ranges from 0 to 10, where 10 is the highest and most punishing setting and 0 is the most forgiving. The HP difficulty setting can be changed alongside other difficulty settings in the [Beatmap Editor](/wiki/Beatmap_Editor). Doing this will change a Ranked beatmap's status to unranked and will erase all local scores on it.
```

Comment:
Para is pretty long, would consider splitting at "The HP difficulty setting can be changed...", since here you're discussing how to change the HP drain, rather than what HP drain _actually_ is.